src/Sequence.js

```diff
@@ -2,6 +2,8 @@
  * Sequence class
  */
 
+var Note = Note || ( require && require('./Note') );
```

Comment:
`require` isn't going to have a binding here for people not using Browserify, so that'll throw a `ReferenceError`.

Any particular reason you don't want to just use the built `TinyMusic.js` file in `/dist`?